test/conftest.py

```diff
@@ -0,0 +1,29 @@
+import playwright_web
+import pytest
+
+
+@pytest.fixture(scope="session")
```

Comment:
To make sure I understand, this runs once per entire test session (all modules). What are we doing to the event loop here?